Sources/ElasticSwift/ElasticSwift.swift

```diff
@@ -16,24 +16,28 @@ public class RestClient: ESClient {
         self.init(settings: Settings.default)
     }
     
-    public func makeGet<T: Codable>() -> GetRequestBuilder<T> {
-        return GetRequestBuilder<T>(withClient: self)
+    public func makeGet<T: Codable>(fromIndex index: String, id: String) -> GetRequestBuilder<T> {
```

Comment:
`ElasticSwift` strictly follows builders pattern. I see the convenience of the proposed change however I see these changes as overload and not as a replacement.